libs/clients/syslumenn/src/clientConfig.json

```diff
@@ -649,6 +785,50 @@
         ]
       }
     },
+    "/v1/AfturkallaAdfarabeidni": {
+      "post": {
+        "tags": ["Syslumenn"],
+        "operationId": "AfturkallaAdfarabeidni_Post",
+        "consumes": [
+          "application/json-patch+json",
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "parameters": [
+          {
+            "name": "skilabod",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AdfarabeidniSimple"
```

Comment:
Note that the OpenAPI specification for the Syslumenn API defines both `Adfarabeidni` and `AdfaraBeidni` (casing of the `b` character), which causes a conflict when generating the client code (`yarn nx run clients-syslumenn:schemas/external-openapi-generator`). Therefor I've temporarily manually renamed `Adfarabeidni` to `AdfarabeidniSimple`, in order to be able to build and feature-deploy this feature. I've already notified the developers of this bug and will revert this change (or simply regenerate this file once it's been addressed using `yarn nx run clients-syslumenn:update-openapi-document`).